data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/TopicConfig.java

```diff
@@ -38,6 +38,11 @@ public class TopicConfig {
     static final Integer DEFAULT_NUM_OF_WORKERS = 2;
     static final Duration DEFAULT_HEART_BEAT_INTERVAL_DURATION = Duration.ofSeconds(5);
 
+
+    private static final Integer DEFAULT_NUM_OF_PARTITIONS = 3;
```

Comment:
pleasae set values as per kafka defaults. if its 3 and then ignore this comment.